.github/workflows/slack.yml

```diff
@@ -4,10 +4,12 @@ on: [push, pull_request, issues]
 
 jobs:
   slack-notifications:
+    continue-on-error: true
     runs-on: ubuntu-20.04
     name: Sends a message to Slack when a push, a pull request or an issue is made
     steps:
       - name: Send message to Slack API
+        continue-on-error: true
```

Comment:
@t8y8 @bcantoni The `continue-on-error` only appears to have an impact if its on the step, and not at the job level.

Reply:
Aww that's right, bad copy/paste job for me then.

Thanks!